xdn-nx-example/XDN.md

```diff
@@ -0,0 +1,216 @@
+# XDN
+
+## Getting Started
+
+To run this Nx example:
+
+```sh
+yarn install
+yarn start
+```
+
+and navigate to [http://127.0.0.1:4200](http://127.0.0.1:4200) with your browser to see the result.
+
+## Building for Deployment
+
+To prepare the project for the XDN and deployment, build the project using:
+
+```sh
+yarn xdn:build
+```
+
+## Simulating a Production Build
+
+To simulate the project on the XDN environment:
+
+```sh
+yarn xdn:run
+```
+
+and navigate to [http://127.0.0.1:3000](http://127.0.0.1:3000) with your browser to see the result.
+
+## Deployment to the XDN
+
+To deploy the project to the XDN:
+
+```sh
+yarn xdn:deploy
+```
+
```

Comment:
@egormesyats can you review these changes? I tried to align it with the guide for the other examples.

Reply:
Looks good!